resources/views/admin/categories.blade.php

```diff
@@ -0,0 +1,85 @@
+                        <i class="fa fa-plus"></i>
+                    </button>
+                </div>
+            </div>
+            <div class="admin-card">
+                <table class="table table-striped table-dark table-hover">
+                    <thead>
+                        <tr>
+                            <th>ID</th>
+                            <th>Title</th>
+                            <th>Created At</th>
+                            <th>Actions</th> <!-- Add a column for actions -->
+                        </tr>
+                        </thead>
+                    <tbody>
+                        @foreach ($categories as $category)
+                            <tr>
+                                <td>{{ $category->id }}</td>
+                                <td>{{ $category->name }}</td>
+                                <td>{{ $category->created_at }}</td>
+                                <td>
+                                    <div class="btn-group">
+                                        <!-- Edit Category Button -->
+                                        <a href="{{ route('admin.editCategory', ['category' => $category->id]) }}" class="btn btn-primary btn-sm">
+                                            <i class="fa fa-pencil"></i> Edit
+                                        </a>
+                                        <!-- Delete Category Button -->
+                                        <form method="POST" action="{{ route('admin.deleteCategory', ['category' => $category]) }}">
+                                            @csrf
+                                            @method('DELETE') <!-- Add this line to set the method to DELETE -->
+                                            <button class="btn btn-danger btn-sm fas fa-trash" type="submit">Delete</button>
+                                        </form>
+                                    </div>
+                                </td>
+                            </tr>
+                        @endforeach
+                    </tbody>
+                </table>
+                {{ $categories->links('pagination::bootstrap-4') }} <!-- Add pagination links at the bottom -->
+            </div>
+            <div class="admin-form" id="newCategoryForm">
+                <div class="form-header">
+                    <span class="form-title">New Category</span>
+                    <button class="close-button" id="closeButton">
+                        <i class="fa fa-times"></i>
+                    </button>
+                </div>
+                <form method="POST" action="{{ route('admin.createCategory') }}">
+                    @csrf
+                    <div class="form-group">
+                        <label for="categoryTitle" class="category-label">Title</label>
+                        <input type="text" name="category_title" class="category-input" id="categoryTitle" placeholder="Title">
```

Comment:
- add old('category_title') to retain old value 
- add error message from request validation